sdk/core/core-xml/src/xml.browser.ts

```diff
@@ -10,6 +10,19 @@ if (!document || !DOMParser || !Node || !XMLSerializer) {
   );
 }
 
+// Policy to make our code Trusted Types compliant at running time.
+//   https://github.com/w3c/webappsec-trusted-types
+// We are calling DOMParser.parseFromString() to parse XML payload from Azure services.
+// The parsed DOM object is not exposed to outside. Scripts are disabled when parsing
+// according to the spec.  There are no HTML/XSS security concerns on the usage of
+// parseFromString() here.
+let ttPolicy: Pick<TrustedTypePolicy, "createHTML"> | undefined;
+if (typeof self.trustedTypes !== "undefined") {
+  ttPolicy = self.trustedTypes.createPolicy("@azure/core-xml#xml.browser", {
```

Comment:
If I'm reading the W3 spec correctly, customers may have to explicitly allow this policy name in their web servers Content-Security-Policy if we use this:

https://github.com/w3c/webappsec-trusted-types/blob/main/explainer.md#limiting-policies

Reply:
Yes, customers would have to explicitly trust our policy, when they enable trusted types in their security policies.